### PR TITLE
1633/handle missing types and functions

### PR DESCRIPTION
**Pull request by longieirl**
Fix for https://github.com/SAP/open-ux-tools/issues/1633

- First draft of changes to support consuming OS modules (btp-utls, axios-exten and store) into Fiori tools.
- Tests added and updated where necessary

### CODE REVIEW

**Review by zdravko-georgiev — DISMISSED** [review]
Types update looks good
Excellent test coverage

File: packages/axios-extension/src/abap/types/adt-types.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ export interface AtoSettings {
     isExtensibilityDevelopmentSystem?: boolean;
     tenantType?: TenantType;
     isTransportRequestRequired?: boolean;
+    isConfigured?: boolean;
```
Comment (IainSAP) [inline packages/axios-extension/src/abap/types/adt-types.ts:89]: Can we add some jsdoc here to explain the properties?

**Review by IainSAP — COMMENTED** [review]
Thanks @longieirl 

- Test cov is good
- One minor comment

File: packages/axios-extension/src/abap/types/adt-types.ts (hunk 1 above)
Comment (IainSAP) [reply packages/axios-extension/src/abap/types/adt-types.ts:98]: Super thx @longieirl !

**Review by IainSAP — APPROVED** [review]
As previous...

**Review by zdravko-georgiev — APPROVED** [review]
Re-approving

**Pull request merged**